src/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
+import * as serviceWorker from './util/serviceWorker';
 
 ReactDOM.render(<App />, document.getElementById('root'));
 
```

Comment:
All the files in folder `src/sw-covers` should belong to public folder. They are just static content.

Reply:
Movie covers moved to static folder.